openembedded-core-mbl/scripts/lib/mbl-signing/signing/pki/endpoints.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# Copyright (c) 2019 Arm Limited and Contributors. All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+
+"""
+Handle the HTTP requests to the endpoints defined in swagger/keystore.yaml.
+
+This should be launched as __main__. It is the front-end of a Flask App.
+
+A server will be spawned at the address given.
+"""
+
+import connexion
+import pathlib
+from signing.pki.backends import signing_backend
+
+
+SPEC_DIR = pathlib.Path(__file__).parent / "swagger"
+
+# Set the signing engine to None as it needs to be instantiated
```

Comment:
We can use the flask `g` object for make this available to the whole flask app. More info: https://flask.palletsprojects.com/en/1.1.x/appcontext/

Reply:
I'll take a look..

Reply:
After discussion with Diego, this is OK for the BETA version, as we're only handling a single HTTP request at a time. I have created https://jira.arm.com/browse/IOTMBL-2418 to cover future API work. I'll also add a comment to the code explaining why we aren't using the flask `g` object.